src/libs/actions/App.js

```diff
@@ -48,15 +47,12 @@ Onyx.connect({
     },
 });
 
-const allPolicies = {};
+let policyIDList = [];
 Onyx.connect({
     key: ONYXKEYS.COLLECTION.POLICY,
-    callback: (val, key) => {
-        if (!val || !key) {
-            return;
-        }
-
-        allPolicies[key] = {...allPolicies[key], ...val};
+    waitForCollectionCallback: true,
+    callback: (policies) => {
+        policyIDList = _.filter(_.pluck(policies, 'id'));
```

Comment:
NAB: If all you're doing is using `_.filter()` to remove falsy values, then I believe `_.compact()` is preferred.

Reply:
TIL! Updated!